src/utils/getApiStaticPath.ts

```diff
@@ -0,0 +1,23 @@
+import { API_CATEGORIES, API_SUB_CATEGORIES } from '../data/api-categories.mjs';
+import { JS_PLATFORMS } from '@/data/platforms';
+
+export const getApiStaticPath = (sub) => {
```

Comment:
Is there a more descriptive name we can use for sub? It's not very clear what this utility accepts.

Is there typing that would also make sense for `sub` and `paths`?